src/pages/Inputs/Inputs.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { useContext } from 'react'
+import ArrowButton from '../../components/Button/ArrowButton'
```

Comment:
We are separating, external libraries from self made components with an line between them, please do that.